app/assets/src/components/common/Heatmap/MetadataLegend.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
```

Comment:
Can you add a comment about this component, since it is now a shared component?

Something like "This component is used on both the AMR and Samples Heatmap. When the user hovers over a label for a metadata field, this component shows a tooltip legend for that field."

It's good to add comments for shared components.